apps/root/src/pages/home/frame/index.tsx

```diff
@@ -73,11 +74,8 @@ const HomeFrame = () => {
                   </StyledContent>
                 </StyledContainer>
               </Grid>
-              <Grid item xs={12} display="flex">
```

Comment:
Is there a reason why we removed this from here and sit it on each component? Would prefer to have the defining structure in this file directly

Reply:
Moved it to the component itself, because it still takes place in the UI when user has no positions, as an empty grid item. I know structure is more clear, but we can handle conditional rendering in the component

Reply:
MM i get that but I would rather keep all structure related to where we display the grids just for clarity